include/Api.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+** EPITECH PROJECT, 2023
+** Visual Studio Live Share (Workspace)
+** File description:
+** ICore
+*/
+
```

Comment:
add ifndef for windows with EXPORT like this: https://github.com/Touxooo/archi-arcade-toulouse/blob/main/include/api.h